src/server.c

```diff
@@ -1685,9 +1685,9 @@ int clientsCronResizeQueryBuffer(client *c) {
     time_t idletime = server.unixtime - c->lastinteraction;
 
     /* There are two conditions to resize the query buffer:
-     * 1) Query buffer is > BIG_ARG and too big for latest peak.
-     * 2) Query buffer is > BIG_ARG and client is idle. */
-    if (querybuf_size > PROTO_MBULK_BIG_ARG &&
+     * 1) Query buffer is > 64k and too big for latest peak.
+     * 2) Query buffer is > 64k and client is idle. */
+    if (querybuf_size > 1024*64 &&
```

Comment:
Should use macro instead of literal constant?

Reply:
Since it can only be used here, I'm not sure if I need to add a macro for it.

Reply:
i think that because this constant is measured against other constants it may be better to declare all at the same place.
i.e. it would make no sense to make it lower than 2*PROTO_IOBUF_LEN.
e.g. even an unforeseen side effect of #5013 caused this line to misbehave.